Breaking change when building from source between 0.5 and 0.11

I get the following message now when upgrading to SDL3 0.11 and building from source:

```
  --- stderr
  CMake Warning at cmake/sdlchecks.cmake:123 (message):
    Unable to find the alsa development library
  Call Stack (most recent call first):
    CMakeLists.txt:1524 (CheckALSA)


  CMake Error at cmake/macros.cmake:400 (message):
    SDL could not find X11 or Wayland development libraries on your system.
    This means SDL will not be able to create windows on a typical unix
    operating system.  Most likely, this is not wanted.

    On Linux, install the packages listed at
    https://github.com/libsdl-org/SDL/blob/main/docs/README-linux.md#build-dependencies


    If you really don't need desktop windows, the documentation tells you how
    to skip this check.
    https://github.com/libsdl-org/SDL/blob/main/docs/README-cmake.md#cmake-fails-to-build-without-x11-or-wayland-support


  Call Stack (most recent call first):
    CMakeLists.txt:3710 (SDL_PrintSummary)


  thread 'main' panicked at /Users/hansl/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cmake-0.1.51/src/lib.rs:1100:5:

  command did not execute successfully, got: exit status: 1
```

A bit of context:

- My target platform does not have X11, and cannot install it (embedded platform)
- I would prefer to statically link SDL3 with my application for it to work (target platform is a very barebone Linux)
- I'm using SDL3 for basically everything else, so this is quite a helpful crate.
- It used to work (building and linking) with SDL3 0.5, so this is something new in the build. Maybe a `sdl3-sys` crate bug, but I don't depend directly with `sdl3-sys` so filing here.
- I'm using `cross-rs` to build, but it shouldn't matter much in this case.

Am I missing a feature flag somewhere where I could disable X11 or work around this bug?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,6 +53,7 @@ static-link = ["sdl3-sys/link-static"]
 link-framework = ["sdl3-sys/link-framework"]
 build-from-source = ["sdl3-sys/build-from-source"]
 build-from-source-static = ["sdl3-sys/build-from-source-static"]
+build-from-source-unix-console = ["sdl3-sys/sdl-unix-console-build"]
 
 unsafe_textures = []
 default = []

